CHANGELOG.md

```diff
@@ -2,6 +2,90 @@
 
 About the past changelog entries, see [old CHANGELOG](CHANGELOG-v4.md) instead.
 
+## Release v5.0.2 - 2023/11/29
+
+### News
+
+* Update fluentd to 1.16.3
+* Update bundled gems
+* msi: support path which contains space or parenthesis (#589)
+* deb: fixed system user/group name in logrotate config (#592,#594)
+  * It fixes a bug that unknown user error was reported.
+* rpm: fixed to create fluentd user as system account (#596)
+  * It fixes a bug that /var/lib/fluent directory was created unexpectedly.
+* rpm: changed to keep system account after removing fluent-package. (#598)
+  * In the previous versions, there was a bug that group was not cleanly removed
+    when the package was upgraded from td-agent v4.
+    This change makes reinstall/downgrade friendly.
+* Update fluent-diagtool to v1.0.3
+  * It supports fluent-package!
+  * It supports Windows partially.
+  * It supports the feature to confirm the manually installed plugin list.
+
+### Core component
+
+* ruby v3.2.2
+* jemalloc v3.6.0
+* OpenSSL 3.1.0 Windows
+* OpenSSL 3.0.8 macOS
+* fluentd v1.16.3 (update)
+
+### Core gems
+
+* async-http v0.60.2
```

Comment:
```suggestion
* async-http v0.61.0
```

Reply:
Thanks. I will fix it after the current CI finishes.

Reply:
Fixed.